Procfile

```diff
@@ -1 +1,2 @@
-web: node lib/main production
+web:                     node lib/main server
+expireCacheBusters: node lib/main expire-cache-busters
```

Comment:
I don't think you need an extra process in the simplest design...
For each workerType we maintain a list of caches to be purged: `[{cacheName: "...", olderThan: new Date()'}, ...]`
When a new cache is purged you append to the list, and if the list is longer than 1000, you drop the oldest entry of the end...

So one entity per workerType is all we need. We can easily store 1000 cache names  and timestamps in a single azure table entity... You can make a test that ensure you don't overflow the size limits, but calling purgeCache 1000 times :)

To ensure that size limits are sane I would limit cacheNames to say 64 bytes or 255bytes... That seems reasonable to me.